book/actix-plugin.md

```diff
@@ -163,15 +139,15 @@ curl http://localhost:8080/api/spec/v2
             }
           }
         }
-      },
-      "parameters": [{
-        "in": "body",
-        "name": "body",
-        "required": true,
-        "schema": {
-          "$ref": "#/definitions/Pet"
-        }
-      }]
+        "parameters": [{
```

Comment:
nit: missing a comma?

Reply:
Yeah, sorry, I was manually changing keys order in JSON to not generate excessive diffs. I'll amend.